### PR TITLE
Update results.py 

### PR DESCRIPTION
**Pull request by srbcheema2**
this PR adds some documentation to the code

### CODE REVIEW

File: resultNITH/results.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 from sys import argv, exit
 
 class Student:
-    #set base year as current year
+    #set base year as current year ie 17 for 2017
     base_year = 17
-    try:
+    try:#you can comment this if not using proxy server
```
Comment (Souravirus) [inline resultNITH/results.py:9]: Please add more documentation

**Comment by Souravirus** [comment]
Thanks for the PR @srbcheema2  
please do the changes

**Comment by Souravirus** [comment]
I am eager to merge it

**Comment by srbcheema2** [comment]
ok @Souravirus

**Comment by srbcheema2** [comment]
done @Souravirus

**Pull request merged**